nupic/regions/CLAClassifierRegion.py

```diff
@@ -256,16 +340,36 @@ def customCompute(self, recordNum, patternNZ, classification):
                    4 : [0.2, 0.4, 0.3, 0.5]}
     """
 
-    return self._claClassifier.compute( recordNum=recordNum,
-                                        patternNZ=patternNZ,
-                                        classification=classification,
-                                        learn = self.learningMode,
-                                        infer = self.inferenceMode)
+    return self._claClassifier.compute(recordNum=recordNum,
+                                       patternNZ=patternNZ,
+                                       classification=classification,
+                                       learn=self.learningMode,
+                                       infer=self.inferenceMode)
 
+  def getOutputValues(self, outputName):
+    """
+    Return the dictionary of output values. Note that these are normal Python
+    lists, rather than numpy arrays. This is to support lists with mixed scalars
+    and strings, as in the case of records with categorical variables
+    """
+    return self._outputValues[outputName]
+
+  def getOutputElementCount(self, name):
```

Comment:
Here you use `name` but in the function above you use `outputName`. If these refer to the same thing then choose one and be consistent.